package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "cesium-unreal",
-  "version": "2.4.0",
+  "version": "2.6.0",
   "lockfileVersion": 3,
   "requires": true,
   "packages": {
     "": {
       "name": "cesium-unreal",
-      "version": "2.4.0",
+      "version": "2.6.0",
```

Comment:
Did you mean to change to this file? 
(it's easy to accidentally commit it)

Reply:
It's automatically changed by npm when running npm install - it seemed fine to commit it, since the version should probably stay up to date anyways, but I agree it's not really related to this PR. I can remove it.